tests/integration/ovs_test.py

```diff
@@ -103,3 +104,16 @@ def test_create_and_remove_ovs_bridge_with_internal_port_and_static_ip():
 
     assertlib.assert_absent(BRIDGE1)
     assertlib.assert_absent(PORT1)
+
+
+@pytest.mark.xfail(
+    raises=NmstateNotImplementedError,
+    reason='OVS link-aggregation is not supported yet',
+    strict=True,
+)
+def test_la_not_implemented():
```

Comment:
Please use `linkaggregation` instead of `la`.

But it does not make sense to test it, and in fact you expect an xfail, which means you are supposed to test the creation of a link-aggregation. Please show that and present how it is supposed to be used.